situp-plugins/common/README.md

```diff
@@ -0,0 +1,34 @@
+# Common SITUP plugins
+
+## `bounded_blocking`
+
+A buffer based off `LinkedBlockingQueue` bounded to the specified capacity. One can read and write records with specified timeout value.
+
+- buffer_size (int): the capacity of the buffer
+- batch_size (int): the reading batch size
```

Comment:
nit: This is the maximum records that will be returned on read with timeout.

Reply:
Changed the description